### PR TITLE
HDS-1793: Set max-width for header

### PR DESCRIPTION
**Pull request by minnas**
## Description

This PR sets max-width for header based on design

## Motivation and Context

[HDS-1793](https://helsinkisolutionoffice.atlassian.net/browse/HDS-1793)

## How Has This Been Tested?

Tested locally

## Demo

Header [In Storybook](https://city-of-helsinki.github.io/hds-demo/HDS-1793-header-max-width/?path=/story/components-header--with-full-features)

[HDS-1793]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-1793?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mrTuomoK — DISMISSED** [review]
Looks ok to me. A bit funny to define the variable on the same level it's being used but I guess that's still good practice :) Good work.

File: packages/react/src/components/header/Header.module.scss [hunk 1]
```diff
@@ -30,6 +30,7 @@
   --action-bar-item-title-font-color: var(--color-black-90);
   --logo-height: 56px;
   --icon-size: 24px;
+  --header-max-width: 1488px;
```
Comment (harriplappalainen) [inline packages/react/src/components/header/Header.module.scss:33]: Isn't the width 1440px? 🤔

Comment (mrTuomoK) [reply packages/react/src/components/header/Header.module.scss:33]: Could you @minnas check and fix?

Comment (minnas) [reply packages/react/src/components/header/Header.module.scss]: 1440px is correct.

**Review by harriplappalainen — APPROVED** [review]
Good job 👍

**Pull request merged**